src/prism.c

```diff
@@ -796,6 +796,27 @@ typedef struct {
     pm_node_t *block;
 } pm_arguments_t;
 
+/**
+ * Retrieve the end location of a `pm_arguments_t` object.
+ */
+static inline const uint8_t *
+arguments_end_location(pm_arguments_t *arguments) {
```

Comment:
I know this is static, but can you call it `pm_arguments_end` anyway?

Reply:
Sure. I was not happy with that name too but couldn't figure out a good one.